### PR TITLE
[FIX JENKINS-33930] adding focus style to delete buttons

### PR DESCRIPTION
**Pull request by gusreiber**
Style fix for button and JS class name adding to box on focus.

@reviewbybees

### CODE REVIEW

File: war/src/main/js/config-scrollspy.js [hunk 1]
```diff
@@ -34,12 +47,13 @@ $(function() {
         });
 
         autoActivateTabs(tabBar);
+        stickTabbar(tabBar);
```
Comment (daniel-beck) [inline war/src/main/js/config-scrollspy.js]: Changes unrelated to this PR.

File: war/src/main/js/config-scrollspy.js [hunk 2]
```diff
@@ -19,9 +19,22 @@ function notify(event) {
...
+    $box.closest('.repeated-chunk').removeClass('hover');
+  }
+}
+
 $(function() {
+  
     var tabBarWidget = require('./widgets/config/tabbar.js');
-
+    var $delete = $('#main-panel')
+      .on('focus','.repeatable-delete button',focusDelete)
```
Comment (kzantow) [inline war/src/main/js/config-scrollspy.js]: FYI, can just .on('focus blur' ...

File: war/src/main/js/config-scrollspy.js [hunk 3]
```diff
@@ -19,9 +19,22 @@ function notify(event) {
     }
 }
 
+function focusDelete(event){
+  var $box = $(this);
+  if(event.type === 'focusin'){
+    $box.closest('.repeated-chunk').addClass('hover');
```
Comment (kzantow) [inline war/src/main/js/config-scrollspy.js]: Why not just keep a button style?

**Comment by kzantow** [comment]
👎 outline around the block is confusing and hard to tell the X actually has focus, just put an outline or outer shadow or something around the X button

**Comment by gusreiber** [comment]
k. I will resubmit with just CSS on the button.

**Pull request closed without merging**